"Install from .ckan" does not catch Krakens

To recreate, make a .ckan file with a spec_version higher than your client and try to install it. The CKAN client closes with no message shown.

Cmdline:

```
CKAN.UnsupportedKraken: KerbalEngineerRedux 1.1.3.0 requires CKAN v1.26, we can't read it.
  at CKAN.CkanModule..ctor (System.String json, CKAN.IGameComparator comparator) [0x00060] in <78d7e82db92c4fa194f1d50cfd9decea>:0 
  at CKAN.CkanModule.FromJson (System.String json) [0x0002a] in <78d7e82db92c4fa194f1d50cfd9decea>:0 
  at CKAN.CkanModule.FromFile (System.String filename) [0x00008] in <78d7e82db92c4fa194f1d50cfd9decea>:0 
  at CKAN.CmdLine.Install.LoadCkanFromFile (CKAN.KSP current_instance, System.String ckan_file) [0x00001] in <78d7e82db92c4fa194f1d50cfd9decea>:0 
  at CKAN.CmdLine.Install.RunCommand (CKAN.KSP ksp, System.Object raw_options) [0x00110] in <78d7e82db92c4fa194f1d50cfd9decea>:0 
  at CKAN.CmdLine.MainClass.RunSimpleAction (CKAN.CmdLine.Options cmdline, CKAN.CmdLine.CommonOptions options, System.String[] args, CKAN.IUser user, CKAN.KSPManager manager) [0x001c0] in <78d7e82db92c4fa194f1d50cfd9decea>:0 
  at CKAN.CmdLine.MainClass.Main (System.String[] args) [0x001df] in <78d7e82db92c4fa194f1d50cfd9decea>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: CKAN.UnsupportedKraken: KerbalEngineerRedux 1.1.3.0 requires CKAN v1.26, we can't read it.
  at CKAN.CkanModule..ctor (System.String json, CKAN.IGameComparator comparator) [0x00060] in <78d7e82db92c4fa194f1d50cfd9decea>:0 
  at CKAN.CkanModule.FromJson (System.String json) [0x0002a] in <78d7e82db92c4fa194f1d50cfd9decea>:0 
  at CKAN.CkanModule.FromFile (System.String filename) [0x00008] in <78d7e82db92c4fa194f1d50cfd9decea>:0 
  at CKAN.CmdLine.Install.LoadCkanFromFile (CKAN.KSP current_instance, System.String ckan_file) [0x00001] in <78d7e82db92c4fa194f1d50cfd9decea>:0 
  at CKAN.CmdLine.Install.RunCommand (CKAN.KSP ksp, System.Object raw_options) [0x00110] in <78d7e82db92c4fa194f1d50cfd9decea>:0 
  at CKAN.CmdLine.MainClass.RunSimpleAction (CKAN.CmdLine.Options cmdline, CKAN.CmdLine.CommonOptions options, System.String[] args, CKAN.IUser user, CKAN.KSPManager manager) [0x001c0] in <78d7e82db92c4fa194f1d50cfd9decea>:0 
  at CKAN.CmdLine.MainClass.Main (System.String[] args) [0x001df] in <78d7e82db92c4fa194f1d50cfd9decea>:0 
```

GUI:

```
System.NullReferenceException: Object reference not set to an instance of an object
  at CKAN.Main.installFromckanToolStripMenuItem_Click (System.Object sender, System.EventArgs e) [0x00052] in <78d7e82db92c4fa194f1d50cfd9decea>:0 
  at System.Windows.Forms.ToolStripItem.OnClick (System.EventArgs e) [0x0001c] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.ToolStripMenuItem.OnClick (System.EventArgs e) [0x000a5] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.ToolStripMenuItem.HandleClick (System.Int32 mouse_clicks, System.EventArgs e) [0x00000] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.ToolStripItem.FireEvent (System.EventArgs e, System.Windows.Forms.ToolStripItemEventType met) [0x00061] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at (wrapper remoting-invoke-with-check) System.Windows.Forms.ToolStripItem:FireEvent (System.EventArgs,System.Windows.Forms.ToolStripItemEventType)
  at System.Windows.Forms.ToolStrip.OnMouseUp (System.Windows.Forms.MouseEventArgs mea) [0x00057] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.ToolStripDropDown.OnMouseUp (System.Windows.Forms.MouseEventArgs mea) [0x00000] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.Control.WmLButtonUp (System.Windows.Forms.Message& m) [0x0007e] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.Control.WndProc (System.Windows.Forms.Message& m) [0x0016f] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.ScrollableControl.WndProc (System.Windows.Forms.Message& m) [0x00000] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.ToolStrip.WndProc (System.Windows.Forms.Message& m) [0x00000] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.ToolStripDropDown.WndProc (System.Windows.Forms.Message& m) [0x0001a] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.Control+ControlWindowTarget.OnMessage (System.Windows.Forms.Message& m) [0x00000] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.Control+ControlNativeWindow.WndProc (System.Windows.Forms.Message& m) [0x0000b] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
  at System.Windows.Forms.NativeWindow.WndProc (System.IntPtr hWnd, System.Windows.Forms.Msg msg, System.IntPtr wParam, System.IntPtr lParam) [0x0008e] in <d5b72d15d4f7424c8a1538e3f19ec2e3>:0 
```

That's because `kraken.InnerException` is null inside this exception handler:

https://github.com/KSP-CKAN/CKAN/blob/81d3f1a7b1ce4b91b25ebfdfffa42efbe997eff7/GUI/Main.cs#L931-L939

Fixing that, we get a proper error:

![image](https://user-images.githubusercontent.com/1559108/33582430-d37e95fe-d91a-11e7-804e-de1b59cd677d.png)

--- a/Cmdline/Action/Install.cs
+++ b/Cmdline/Action/Install.cs
@@ -65,7 +65,16 @@ namespace CKAN.CmdLine
                     }
 
                     // Parse the JSON file.
-                    options.modules.Add(LoadCkanFromFile(ksp, filename).identifier);
+                    try
+                    {
+                        options.modules.Add(LoadCkanFromFile(ksp, filename).identifier);
+                    }
+                    catch (Kraken kraken)
+                    {
+                        user.RaiseError(kraken.InnerException == null
+                            ? kraken.Message
+                            : $"{kraken.Message}: {kraken.InnerException.Message}");
+                    }
                 }
 
                 // At times RunCommand() calls itself recursively - in this case we do

--- a/GUI/Main.cs
+++ b/GUI/Main.cs
@@ -934,7 +934,9 @@ namespace CKAN
                 }
                 catch (Kraken kraken)
                 {
-                    currentUser.RaiseError(kraken.Message + ": " + kraken.InnerException.Message);
+                    currentUser.RaiseError(kraken.InnerException == null
+                        ? kraken.Message
+                        : $"{kraken.Message}: {kraken.InnerException.Message}");
                     return;
                 }
                 catch (Exception ex)

